### PR TITLE
Refactor for new data interactions, seed script

### PR DESCRIPTION
**Pull request by azdoolittle**

### CODE REVIEW

**Comment by azdoolittle** [comment]
> Nice markup. If its not too much I would say putting emojis (next to headers), and having pics or gifs can help make your readme stand out. This is more in the perspective of when employers look through repos on github. Having the pic of gif about your app will say it all before they even read (especially when put somewhere towards the top).

@Nguyephi Yeah, the gif preview is a good idea. I was planning to do that when I've finalized the front end

**Comment by Nguyephi** [comment]
lgtm

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import styles from './styles.css';
-import App from './components/App.jsx';
+import ChannelService from './components/ChannelService.jsx';
 
-ReactDOM.render(<App />, document.getElementById('app'));
+let id = window.location.search.slice(1);
+let videoId = id ? id : 1;
```
Comment (Kraig5) [inline client/src/index.jsx]: Nice

File: db/seed.js [hunk 2]
```diff
@@ -4,32 +4,88 @@
...
-  channel.title = faker.random.words();
-  channel.viewer_count = faker.random.number({
-    min: 1000,
-    max: 15000,
-  });
-  channel.thumbnail_url = faker.internet.avatar();
-  return channel;
-};
+//top streaming games in order according to Twitch
+const gameName = [
```
Comment (priscilagonzalez) [inline db/seed.js]: All the game names! I only used Fortnite. Nice!

Comment (azdoolittle) [reply db/seed.js]: @priscilagonzalez feel free to use them if you need more unique data

Comment (priscilagonzalez) [reply db/seed.js]: Thanks! :)

**Pull request merged**